services/manager/lib/io/config/index.js

```diff
@@ -0,0 +1,18 @@
+const { join, resolve } = require('path');
+const { promisify } = require('util');
+const fs = require('fs');
+
+const readFile = promisify(fs.readFile);
+
+const configPath = process.env.CONFIG_PATH
+  ? resolve(process.env.CONFIG_PATH)
+  : join(__dirname, '../../../../../env.json');
+
+module.exports = async () => {
+  try {
+    return JSON.parse(await readFile(configPath));
+  } catch (error) {
+    console.error('Error reading config: ', error);
+    return null;
```

Comment:
Maybe this sounds always return an array? I'd would make the recieving function similar and they'd be no edge cases

Reply:
*should* always return an array.

Reply:
Makes sense, thanks!